src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -5492,8 +5615,20 @@ private void logWarning(String message, Exception e) {
     
     //main() method for testing only
     public static void main (String [] args) {
-        JFrame frame = new JFrame();
-        frame.getContentPane().add(new EditProblemPane());
+
+        //build an EditProblemFrame containing an EditProblemPane containing an EditProblemSandboxPane
+        EditProblemFrame frame = new EditProblemFrame();
+        
+        //put a contest and controller in the frame (which also puts it in the panes)
+        IInternalContest contest;
+        IInternalController controller;
+
+        SampleContest sampleContest = new SampleContest();
```

Comment:
BUILD FAILURE.   This cause a build failure, SampleContest is not part of the src/ tree it is part of the test/ tree.
Error is: symbol:   class SampleContest
[javac]   location: class EditProblemPane

Reply:
Hmmm.... I'm not clear what the correct fix for this is.   Can you suggest a fix?   Or shall I just remove the test function (main)?

Reply:
I've commented out the offending code and will push an update.